docs/rules/valid-local-dependency.md

```diff
@@ -0,0 +1,25 @@
+# Ensures that local dependencies specified in a package.json exist
+
+## Rule Details
+
+This rule validates the casing for `file:` and `link:` dependencies in a `package.json` file. If the casing of a `link:` or `file:` path is incorrect yarn does not sub out the link when releasing.
```

Comment:
> yarn

[Thoughts] I wonder what pnpm does here? And bun? Deno? I haven't looked into them and don't have plans to. 😄

Reply:
🤔 good point -- I could remove the yarn explanation all together to generalize the rule. I originally wrote this because I kept messing up casing in my `yarn` project.